internal/controller/redis_connector.go

```diff
@@ -70,6 +70,7 @@ func UnregisterWithRedis(client *redis.Client, account, nodeID, hostname string)
 	_, err := client.TxPipelined(func(pipe redis.Pipeliner) error {
 		client.Del(getConnectionKey(account, nodeID))
 		removeIndexes(client, account, nodeID, hostname)
+		metrics.unregisterStaleConnectionFromRedis.Inc()
```

Comment:
This method is used in the case of normal un-registration so I don't think this is the right place for this counter.